### PR TITLE
Add ability to filter keys

### PR DESCRIPTION
**Pull request by rmnbrd**

### CODE REVIEW

**Comment by rmnbrd** [comment]
Sorry for this huge PR 🤷‍♂️

File: src/containers/KeyItemContainer/index.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ type Props = {
 }
 
 const KeyItemContainer = ({ projectSlug, router, ...props }: Props) => {
-  const page = router.query.page || 0
+  const page = Number(router.query.page - 1) || 0
```
Comment (Tom-Bonnike) [inline src/containers/KeyItemContainer/index.js]: should probably be: `Number(router.query.page) - 1 || 0`

File: src/containers/NewKeyModalContainer/index.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ type Props = {
 }
 
 const NewKeyModalContainer = ({ getDialog, project, router }: Props) => {
-  const page = router.query.page || 0
+  const page = Number(router.query.page - 1) || 0
```
Comment (Tom-Bonnike) [inline src/containers/NewKeyModalContainer/index.js]: Same here

File: src/containers/TranslationValueRowContainer/index.js [hunk 3]
```diff
@@ -40,8 +40,9 @@ const TranslationValueRowContainer = ({
           query: projectQuery,
           variables: {
             projectSlug: router.query.projectSlug,
-            page: router.query.page || 0,
-            pageSize: KEYS_PER_PAGE
+            page: Number(router.query.page - 1) || 0,
```
Comment (Tom-Bonnike) [inline src/containers/TranslationValueRowContainer/index.js]: Same here

**Pull request merged**